spec/tap_installer_spec.rb

```diff
@@ -13,21 +17,36 @@ def do_install(clone_target = nil)
     end
   end
 
+  context ".pinned_installed_taps" do
+    it "calls Homebrew" do
+      Bundle::TapInstaller.pinned_installed_taps
```

Comment:
I can tell that you're following suit from `context ".installed_taps"` above, but I feel like this should have some kind of explicit expectation (as should the one above). I don't feel strongly enough about it to block on that feeling, though.